docs/specification.md

```diff
@@ -0,0 +1,169 @@
+
+#### Identity
+
+The next step will be the identification ourselves in the network. First of all lets generate an 
+identity using `tezos-node identity generate` command. It will take a short amount of time. 
+After the end, you will be able to see your identity here `~/.tezos-node/identity.json`.
+
+#### Account
+
+Before we will start to use the test net we need to set up the node for `tezos-client`.
+With `tezos-client -A <IP addr|host> -P <port number> config update` you can easily add it to
+`tezos-client` config. For example, you can use **Serokell** node: 
+`delphi.testnet.tezos.serokell.team:8732`
+
+Now we can start to interact with Tezos test net, but we still have no account with money. So, lets 
+get it:
+* Navigate to https://faucet.tzalpha.net and download the file. This file will contains all
+the required information for account activation.
+* Activate account using 
+`tezos-client activate account <account_alias> with <path_to_the_downloaded_file>` command
+* After that you will be able to see your address in the net. Lets open 
+[block explorer](https://tzkt.io/), choose `carthage` net and search for the address.
+
+One more thing you need to do before swap test is to create the account with whom the swap will be 
+produced:
+* Generate new account `tezos-client gen keys bob`
+* And transfer some money to him `tezos-client transfer 1000 from alice to bob`
+* You will see error about burn cap and needed quantity to execute this operation.
+* Add ` --burn-cap <burn_cap_number_from_error message>` to the previous command and try again.
+
+You can easily see all addresses using `tezos-client list known addresses`
+
+#### Originate the contract
+
+After creating identity and accounts we can originate our bridge contract. You can find contract
+code [here](../contract/morley-contract/contract.tz) with 
+[storage](../contract/morley-contract/storage.tz) example. Lets originate this contract and add
+previously created accounts to its storage (you alos may need to add a --burn-cap flag to this 
+command):
+
+```
+tezos-client originate contract blndOnTezos transferring 1 from alice running <path_to_contract> \
+  --init 'Pair (Pair (Pair {Elt <alice_addres> 1000; Elt <bob_address> 1000} { }) \
+  (Pair <alice_addres> (Pair False 2000))) (Pair { } { })'
+```
+
+This command will originate bridge contract with alias `blndOnTezos`, two accounts holding 
+1000 tokens and `alice` as administrator. Also in the command output you can see contract address 
+and find it in the [block explorer](https://better-call.dev/).
+
+Now you are ready to interact with Tezos part of `Bridge` contract.
+
+### Ethereum
+
+#### Build
+
+First clone [Blend](https://github.com/StakerDAO/blend) repository, it contains all needed
+things to build, deploy and interact with Bridge contract. To build it you need to have Node v14 and
+install all with `npm install`. Then build and compile contracts with `npm run build` and 
+`npx oz compile`.
```

Comment:
AFAIR this is not technically true. With `npm run build` you are building CLI (_ts_ -> _js_) and `npx oz compile` is one which actually compiles contracts